python/aida_interchange/aida_rdf_ontologies.py

```diff
@@ -38,11 +37,8 @@
            "Physical.LocatedNear", "Physical.OrganizationHeadquarter", "Physical.OrganizationLocationOrigin",
            "Physical.Resident", "Weapon"])
 
-"""
-The domain ontology.
 
-For the moment, this is hard-coded to match ColdStart
-"""
+# The domain ontology. For the moment, this is hard-coded to match ColdStart
```

Comment:
This comment does not show up when generating the documentation. Was that intended? If you want to it show up append `:` after the hash.

Reply:
Yes - intentional.  I removed the comment that was getting generated in the incorrect location and made the decision that these comments didn't need to be in the python api docs.  It's not really part of the API.